log.go

```diff
@@ -91,8 +91,10 @@ var (
 
 // genSubLogger creates a logger for a subsystem. We provide an instance of
 // a signal.Interceptor to be able to shutdown in the case of a critical error.
-func genSubLogger(root *build.RotatingLogWriter,
-	interceptor signal.Interceptor) func(string) btclog.Logger {
+func genSubLogger(
```

Comment:
Please don't re-format the code here. These lines are perfectly well formatted according to our [formatting guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md)